domain/entity/latest_entry.go

```diff
@@ -0,0 +1,90 @@
+package entity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	LatestEntryCodeTamai    = "tamai-sd"
+	LatestEntryCodeMomota   = "momota-sd"
+	LatestEntryCodeAriyasu  = "ariyasu-sd"
+	LatestEntryCodeSasaki   = "sasaki-sd"
+	LatestEntryCodeTakagi   = "takagi-sd"
+	LatestEntryCodeHappyclo = "happyclo"
+)
+
+type (
+	LatestEntry struct {
+		ID        string `datastore:"-" goon:"id"`
+		Code      string
+		URL       string
+		CreatedAt time.Time
+		UpdatedAt time.Time
+	}
+
+	LatestEntryRepository interface {
+		Save(context.Context, *LatestEntry) error
+		GetTamaiURL(context.Context) string
+		GetMomotaURL(context.Context) string
+		GetAriyasuURL(context.Context) string
+		GetSasakiURL(context.Context) string
+		GetTakagiURL(context.Context) string
+		GetHappycloURL(context.Context) string
+	}
+)
+
+func (l *LatestEntry) SetCreatedAt(t time.Time) {
+	l.CreatedAt = t
+}
+
+func (l *LatestEntry) GetCreatedAt() time.Time {
+	return l.CreatedAt
+}
+
+func (l *LatestEntry) SetUpdatedAt(t time.Time) {
+	l.UpdatedAt = t
+}
+
+func (l *LatestEntry) Load(p []datastore.Property) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method LatestEntry.Load should have comment or be unexported